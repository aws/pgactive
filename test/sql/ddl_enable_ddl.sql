SET bdr.skip_ddl_replication = true;
CREATE TABLE should_ok ( id integer );

RESET bdr.skip_ddl_replication;
CREATE TABLE create_ok (id integer);

SET bdr.skip_ddl_replication = true;
ALTER TABLE create_ok ADD COLUMN alter_should_ok text;

RESET bdr.skip_ddl_replication;
DROP TABLE create_ok;

-- Now for the rest of the DDL tests, presume they're allowed,
-- otherwise they'll get pointlessly verbose.
ALTER DATABASE regression RESET bdr.skip_ddl_replication;
ALTER DATABASE postgres RESET bdr.skip_ddl_replication;
