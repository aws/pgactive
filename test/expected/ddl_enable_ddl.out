SET pgactive.skip_ddl_replication = true;
CREATE TABLE should_ok ( id integer );
RESET pgactive.skip_ddl_replication;
CREATE TABLE create_ok (id integer);
SET pgactive.skip_ddl_replication = true;
ALTER TABLE create_ok ADD COLUMN alter_should_ok text;
RESET pgactive.skip_ddl_replication;
DROP TABLE create_ok;
-- Now for the rest of the DDL tests, presume they're allowed,
-- otherwise they'll get pointlessly verbose.
ALTER DATABASE regression RESET pgactive.skip_ddl_replication;
ALTER DATABASE postgres RESET pgactive.skip_ddl_replication;
